### PR TITLE
Have the network instance subnet ip rule created as part of NAT activ…

### PR DESCRIPTION
**Pull request by gkodali-zededa**
…ation instead of relying on PBR.

Signed-off-by: Gopi krishna Kodali <gkodali@zededa.com>

### CODE REVIEW

File: pkg/pillar/cmd/zedrouter/pbr.go [hunk 1]
```diff
@@ -253,15 +253,11 @@ func PbrAddrChange(deviceNetworkStatus *types.DeviceNetworkStatus,
 		changed = devicenetwork.IfindexToAddrsAdd(log, change.LinkIndex,
```
Comment (eriknordmark) [inline pkg/pillar/cmd/zedrouter/pbr.go:253]: This whole section of code is now a no-op. Perhaps even the function is a no-op snd can be removed?

**Review by eriknordmark — COMMENTED** [review]
Looks like more code can be removed

File: pkg/pillar/cmd/zedrouter/pbr.go (hunk 1 above)
Comment (gkodali-zededa) [reply pkg/pillar/cmd/zedrouter/pbr.go:253]: Yes. I think this function can be removed. What this function does is already done by AddrChange function called from NIM context.

**Review by rvs — APPROVED** [review]
Looks awesome -- I love code being removed! ;-) LGTM

**Review by eriknordmark — APPROVED** [review]
Great

**Pull request merged**